docs/abtest.rst

```diff
@@ -8,6 +8,39 @@
 A/B Testing
 ===========
 
+Nimbus experiments
+------------------
+
+Nimbus (Experimenter), is Mozilla's 1st-party experimentation platform.
+Experiments from this platform will split traffic before users arrive on
+the page (meaning no client-side JS delay).
+
+.. Note::
+
+    Bedrock code will still determine what templates to show based on locales
+    and the url parameters added by Nimbus.
+
+There are generally 3 user groups to consider:
+
+#. In experiment - control branch
+#. In experiment - treatment branch
+#. Not in experiment
+
+The Firefox team will set up the experiment and provide the URL params:
+
+- **branch** (required): identifies an experiment. When there is a single experiment, the branch value can be used to distinguish control and treatment (i.e. ``experiment-wnp-136-control`` and ``experiment-wnp-136-vpn``)
```

Comment:
I think we have different understandings of this term between Nimbus & Bedrock

in Nimbus, it's a branch that could be control, treatment-a, treatment-b, etc.

in Bedrock, it's an experiment identifier

Reply:
This might be correct, but it sounds confusing to me as bedrock has no idea if there is a single experiment running or not. IMHO it would be clearer if we always specified an experiment branch and two or more variations?

Reply:
Confirmation from Nimbus team side that we can standardize to use both `branch` and `variant` urls going forward